dotnet/Services/ProductReviewService.cs

```diff
@@ -450,7 +450,7 @@ public async Task<ReviewsResponseWrapper> GetReviewsByProductId(string productId
                         searchQuery = $"{searchQuery}&approved=true";
                     }
 
-                    wrapper = await this._productReviewRepository.GetProductReviewsMD($"productId={productId}{sort}{searchQuery}{ratingQuery}{localeQuery}", from.ToString(), to.ToString());
+                    wrapper = await this._productReviewRepository.GetProductReviewsMD($"productId={productId}{sort}{searchQuery}{ratingQuery}&{localeQuery}", from.ToString(), to.ToString());
```

Comment:
Hmm, I think `localeQuery` needs to be inside a `_where=` statement, similar to line 440 (not to mention the other variables).

```suggestion
                    wrapper = await this._productReviewRepository.GetProductReviewsMD($"_where=(productId={productId}){searchQuery}{ratingQuery}{localeQuery}{sort}", from.ToString(), to.ToString());
```

Reply:
Previously this method had if (!string.IsNullOrEmpty(locale))
                    {
                        localeQuery = $"&locale={locale}-*";
                    }

Reply:
Look at 
#254